_posts/2018-11-27-no66-every-hack-is-a-target.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Berlin Hack & Tell \#66 - Every Hack is a Target
+date: 2018-11-27
+time: '18:30'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/256569672/
+---
+
+* [GitHub Notifier for Mac OS](https://github.com/sargsyan/github-notifier) by [sargsyan](https://github.com/sargsyan)
+* [Ethereum Auth Chat](https://github.com/okwme/eac-chat) by [billy rennekamp](https://github.com/okwme)
+* [RPi Visual Echo and Zoom](http://www.fhack.org/2018/11/27/visual-echo-on-raspberry-pi/) by [Fhain hacks](fhack.org)
+* [phonk Android Web IDE](http://www.phonk.io) by [Víctor Díaz Barrales](http://www.victordiazbarrales.com/) - **Hack of the month**
+* [Lil' Trump](https://github.com/bananatron/lil-trump) by [Spenser Coke](https://spenser.xyz/)
+* [Audio Shop](https://github.com/robertfoss/audio_shop) by [Robert Foss](https://memcpy.io/)
+* [Horse Del Ray](https://github.com/roryscarson/Horse_Del_Rey) by [Rory Carson](https://github.com/roryscarson)
+* [incook](https://git.schulz.codes/open-source/incook) by [Daniel Schulz](https://schulz.codes/)
+
+Overflow:
```

Comment:
I was curious about this? Does this mean there were 9 slots that evening?

Reply:
yes - the crowd wanted it. I initially though "no" - but did not want to be a dictator and asked in the room - most people wanted to see this 9th talk (and it happily did not crash the system)

Reply:
@ligi That's cool, didn't want to come down on you as evil overlord. We had reasons to create the slot system and it made the event better, more predictable and manageable. However, it should not be considered an unchangable system. In fact, I'm quite happy to see that level of ownership. 👍

Reply:
The audience support was overwhelming and since we missed a month and were about to skip a month, making an exception seemed reasonable to me. And like @ligi said, I was able to add a slot to the voting system without issue!